src/Application/Actions/Donations/Create.php

```diff
@@ -56,9 +56,6 @@ protected function action(Request $request, Response $response, array $args): Re
 
         $body = (string) $request->getBody();
 
-        // Frontend just has to send this temporarily so we know when its switched over from the card element to the new payment element.
-        $forNewPaymentElement = ($request->getQueryParams()['forNewPaymentElement'] ?? '') === 'true';
```

Comment:
I've tested with the current and new versions of the frontend - the current version of frontend works with the new style of payment intents, the new one doesn't work with the old style. So we can just always generate new style ones in matchbot and it will be compatible with both.